plutus-benchmark/README.md

```diff
@@ -35,9 +35,9 @@ This directory contains two sets of benchmarks:
 * `validation`:  a number of Plutus Core scripts extracted from the `plutus-use-cases` tests which represent realistic on-chain
    transaction validations.
 
-   * The scripts are stored as Plutus Core source in `validation/data`, along with a description
-     of how to combine them to obtain executable applied validators.
-   * Benchmarking code is stored in `validation/Main.hs`.
+   * The scripts are stored in flat-encoded form in various subdirectories of `validation/data`.
```

Comment:
i.e. "run `cabal run plutus-use-cases:plutus-use-cases-scripts -- plutus-benchmark/validation/data`"

Reply:
Ah... but of course that won't work since you've got this directory structure! It would be nice if there was _one_ command that regenerated them. Ideas:
- Teach the generator to put them in directories?
- Arrange the benchmark groups by taking everything in the name up to the first `-`?